src/components/Messages/MessageFolder.jsx

```diff
@@ -4,9 +4,12 @@ import ReactPaginate from 'react-paginate';
 // Styling Imports
```

Comment:
Think we can just group up all the MUI Imports together and remove `Styling Imports`

Reply:
Changed!

Sorry, I missed this comment earlier.